pennylane_sphinx_theme/_version.py

```diff
@@ -3,4 +3,4 @@
 using the following format: <major>.<minor>.<patch>[-<pre-release>].
 """
 
-__version__ = "0.6.0-dev"
+__version__ = "0.6.0"
```

Comment:
Set Python package version to v0.5.6.
```suggestion
__version__ = "0.5.6"
```